source/particle/generator/reference_cell.cc

```diff
@@ -131,10 +126,12 @@ namespace aspect
             {
               prm.enter_subsection("Reference cell");
               {
-                prm.declare_entry ("Number of tracers per cell per direction", "10",
-                                   Patterns::Double (0),
-                                   "Total number of tracers to create for each cell for each spatial dimension."
-                                   "The number is parsed as a floating point number (so that one can "
+                prm.declare_entry ("Number of tracers per cell per direction", "2",
+                                   Patterns::List(Patterns::Double(0)),
+                                   "List of number of tracers to create per cell and spatial dimension."
+                                   "The size of the list is the number of spatial dimensions. If only"
+                                   "one value is given, then each spatial dimension is set to the same value."
+                                   "The list of numbers are parsed as a floating point number (so that one can "
                                    "specify, for example, '1e4' particles) but it is interpreted as "
```

Comment:
I hope nobody tries 1e4 particles per direction in a model with 1e6 cells in 3D :smile:. That would be 1e18 particles. But keep it as an example.